local-volume/bootstrapper/bootstrapper.go

```diff
@@ -250,6 +250,22 @@ func createDaemonSet(client *kubernetes.Clientset, namespace string, config map[
 			MountPath: mount.MountDir,
 		})
 	}
+	// Appending configMap as a volume
+	volumes = append(volumes, v1.Volume{
+		Name: *volumeConfigName,
+		VolumeSource: v1.VolumeSource{
+			ConfigMap: &v1.ConfigMapVolumeSource{
+				LocalObjectReference: v1.LocalObjectReference{
+					Name: *volumeConfigName,
+				},
+			},
+		},
+	})
+	// Appending a mount point for ConfigMap Volume
+	volumeMounts = append(volumeMounts, v1.VolumeMount{
+		Name:      *volumeConfigName,
+		MountPath: "/etc/provisioner/config",
```

Comment:
Should MountPath reference the provisionerConfigPath variable in common.go?

Reply:
yes, correct I will replace it with constant defined in common.